salt/tops/varstack.py

```diff
@@ -45,28 +45,19 @@
 
 from __future__ import absolute_import, print_function, unicode_literals
 
-# Import python libs
-import logging
-
 try:
-    HAS_VARSTACK = False
     import varstack
-    HAS_VARSTACK = True
 except ImportError:
-    pass
-
-# Set up logging
-log = logging.getLogger(__name__)
+    varstack = None
 
 # Define the module's virtual name
 __virtualname__ = 'varstack'
```

Comment:
You need this only in case you want to rename the module and join it to some other group of those, like `pkg` for example. Otherwise [keep it simple](https://en.wikipedia.org/wiki/KISS_principle).

Reply:
We should probably document a standard. There is a mix of both styles at present.